featureFlags/README.md

```diff
@@ -0,0 +1,163 @@
+# FeatureFlags
+
+# FeatureFlag object
+
+```
+{
+    'name': string,
+    'title':string,
+    'created_at': timestamp,
+    'updated_at': timestamp,
+    'config': {'enabled': boolean},
+    'owner': {
```

Comment:
Keep feature object as it was.

Only add the owner as an object in GET route response.